datasets/meldingen/meldingen/v1.0.0.json

```diff
@@ -0,0 +1,233 @@
+{
+    "id": "meldingen",
+    "type": "table",
+    "description": "Open dataset over meldingen openbare ruimte Amsterdam (MORA). In deze dataset zit een subset van de de meldingsgegevens van alle meldingen vanaf medio 2018. Hierin zijn de volgende hoofdcategorieën meegenomen: Afval, Civiele Constructies, Openbaar groen en water, Overlast Bedrijven en Horeca, Overlast in de openbare ruimte, Overlast van boten, Overlast van dieren, Overlast van en door personen of groepen, Schoon, Wegen, verkeer, straatmeubilair. De volgende type meldingen zijn uitgesloten in deze dataset:  Meldingen van de overige hoofdcategorieen en meldingen die na binnenkomst zijn gesplitst. Van gesplitste meldingen worden alleen de afgesplitste meldingen meegenomen (dus niet de initiële melding die na splitsing direct gesloten wordt). Gesplitste / afgesplitst is alleen van toepassing op meldingen tot 18 november 2020",
+    "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "mainGeometry": "geometrie",
```

Comment:
Ik zie geen `geometrie` veld in deze tabel?

Reply:
Ja klopt, wel opvallend. Blijkbaar alleen meldingenbuurt. Zo heb ik dat begrepen van datateam omgevingswet.